client/state/data-layer/wpcom/concierge/schedules/appointments/detail/from-api.js

```diff
@@ -13,4 +13,4 @@ export const transform = ( { begin_timestamp, end_timestamp, schedule_id, ...res
 	...rest,
 } );
 
-export default makeJsonSchemaParser( responseSchema, transform );
+export default makeJsonSchemaParser( responseSchema.properties.next_appointment, transform );
```

Comment:
I know @nb asked about re-using Schemas, but this feels a little off to me. This file is the canonical endpoint that represents "details for a single appointment", but since it's pulling its schema from `initial.next_appointment` it's treating that schema as the canonical representation for single appointment data.

Doing the reverse feels saner to me (keeping the `appointments/details/schema.json` and that becomes the canonical representation that's imported into the `initial` endpoint handler). But I even wonder if there's a better way to represent schemas and transforms for these shared entities in some higher level, so that instead of needing to know "which endpoint has the canonical Appointment schema?" there's a clear shared schema repository for all Concierge endpoints...

@nb thoughts on this?